File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/livestream/state/BeautyState.java
package com.trtc.uikit.livekit.livestream.state;

import com.trtc.tuikit.common.livedata.LiveData;

public class BeautyState {
    public LiveData<Integer> smoothLevel         = new LiveData<>(0);
    public LiveData<Integer> whitenessLevel      = new LiveData<>(0);
    public LiveData<Integer> ruddyLevel          = new LiveData<>(0);
    public LiveData<Boolean> glContextCreateFlag = new LiveData<>();


    public void reset() {
        smoothLevel.set(0);
        whitenessLevel.set(0);
        ruddyLevel.set(0);
        glContextCreateFlag.set(null);
    }
}

File: Flutter/livekit/lib/component/beauty/state/beauty_state.dart
import 'package:flutter/material.dart';

class BeautyState {
  static const int defaultSmoothLevel = 0;
  static const int defaultWhitenessLevel = 0;
  static const int defaultRuddyLevel = 0;

  final ValueNotifier<int> smoothLevel = ValueNotifier<int>(defaultSmoothLevel);
  final ValueNotifier<int> whitenessLevel = ValueNotifier<int>(defaultWhitenessLevel);
  final ValueNotifier<int> ruddyLevel = ValueNotifier<int>(defaultRuddyLevel);

  void reset() {
    smoothLevel.value = defaultSmoothLevel;
    whitenessLevel.value = defaultWhitenessLevel;
    ruddyLevel.value = defaultRuddyLevel;
  }
}
